tests/e2e/local_interactive_sdk_oauth_test.py

```diff
@@ -46,9 +44,8 @@ def run_local_interactives(self):
                 num_workers=1,
                 worker_cpu_requests=1,
                 worker_cpu_limits=1,
-                worker_memory_requests=4,
+                worker_memory_requests=1,
```

Comment:
Is this correct change?

Reply:
@sutaakar Yes, I have tested this configuration with 1 memory requests for local-interactive test, worked seamlessly! 

![image](https://github.com/user-attachments/assets/97cb7291-2133-4126-8bab-78fed51b9146)

Reply:
Oh, ok
I though it was a limit. Thanks